query-languages/dax/len-function-dax.md

```diff
@@ -32,6 +32,10 @@ A whole number indicating the number of characters in the text string.
 
 - If you use LEN with a column that contains non-text values, such as dates or Booleans, the function implicitly casts the value to text, using the current column format.
 
+- Starting with desktop january release, user can set new model property UnicodeCharacterBehavior. Depending on value of the property, the output of LEN(<text>) function changes.
```

Comment:
I'd recommend we don't make it so dependent on the january release in the docs. I'd just say: In recent versions, you can set the UnicodeCharacterBehavior property.